benchmarks/TerraFX.Interop.Mimalloc.AllocatorsBenchmarks.csproj

```diff
@@ -0,0 +1,27 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
+    <NoWarn>
+      $(NoWarn);
+
+      <!--Missing XML comments-->
+      CS1591
+    </NoWarn>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Include="BenchmarkResultsInterpreted.md" />
+    <None Include="BenchmarkResultsRaw.md" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="BenchmarkDotNet" Version="0.13.1" />
+    <PackageReference Include="TerraFX.Interop.Mimalloc" Version="1.6.0-beta2" />
```

Comment:
This shouldn't be a package reference, it should directly reference the project instead.